### ISSUE TITLE
Mentor Registration for January 30th 2016

### ISSUE THREAD

**Issue by jedireza** [nodeschool/sanfrancisco#108]
Mozilla is hosting us on January 30th from 1pm - 5pm. The address is: 2 Harrison St, San Francisco, CA 94105

We're accepting up to 70 attendees for this workshop. So we'll need about 10-ish mentors.

Mentors should commit to completing (or have already completed) the [core NodeSchool workshoppers](http://nodeschool.io/#workshopper-list) before the event. Having also completed elective workshoppers is a great bonus. We also encourage mentors to review the [Mentor Best Practices](https://github.com/nodeschool/organizers/wiki/Event-Mentor-Best-Practices) wiki page.

Comment to get involved or ask questions.

Confirmed mentors:

 - [x] @TheAlphaNerd
 - [x] @Trott
 - [x] @paulgrock
 - [x] @noffle
 - [x] @isurfbecause
 - [x] @Shinobi881
 - [x] @Fauntleroy
 - [x] @cachrisman
 - [X] @negomi
 - [x] @jeffcarp

Organizers and volunteers:

 - [x] @jedireza
 - [x] @llkats

**Comment by TheAlphaNerd** [comment]
o/

**Comment by jedireza** [comment]
@TheAlphaNerd thanks! :shipit:

**Comment by Trott** [comment]
In!

**Comment by paulgrock** [comment]
In

**Comment by noffle** [comment]
Interested and happy to help!

**Comment by isurfbecause** [comment]
You know I'm in

**Comment by Shinobi881** [comment]
I like to help!

**Comment by jedireza** [comment]
@Trott @paulgrock @noffle @isurfbecause @Shinobi881

Big thanks! :pray: :+1: :raised_hands: :surfer: :v:

**Comment by Fauntleroy** [comment]
I'll be there. Feel free to omit me from the official mentor list if you have some more diverse candidates, though.

**Comment by jedireza** [comment]
@Fauntleroy thanks :pizza:

**Comment by cachrisman** [comment]
I'd love to be a mentor!

--
Charlie Chrisman
303-902-5082

**Comment by jedireza** [comment]
@cachrisman thank you :tada:

**Comment by negomi** [comment]
I'm also in if needed! :cat:

**Comment by jedireza** [comment]
@negomi thank you :smile_cat:

**Comment by llkats** [comment]
I am tentatively in, probably keeping a low profile at this event. Let me know if there's any logistical stuff I can pitch in on before the day. :smile:

**Comment by jedireza** [comment]
@llkats yay :cat2:

**Comment by jeffcarp** [comment]
Hello! I haven't been involved with NodeSchool before but I am interested in mentoring if needed.

**Comment by jedireza** [comment]
@jeffcarp welcome! Thanks for volunteering. I've added you to the list. Have you attended a NodeSchool event before? If not, that's fine. This can be your first. :+1:

**Comment by jeffcarp** [comment]
@jedireza thank you! I have not attended one before. Is there anything else I need to know? I will complete the core + read the mentor guide before next Saturday.

**Comment by jedireza** [comment]
Completing the core workshoppers and reading the mentor guide is highly recommended. The events are informal. It's like we're all hanging out in a big living room. As people make their way through the workshoppers they can ask for help to get unblocked.

**Comment by isurfbecause** [comment]
@jedireza @Fauntleroy Just wondering if we have a slack channel? It would be awesome to keep in contact with other mentors and participants. It would be a great way to send quick updates to mentors. I find myself also wondering how others I helped are doing after the events.

**Comment by jedireza** [comment]
To my knowledge SF doesn't have a Slack channel. I'd hope we could use GitHub issues for most communication. We can also take advantange of the @nodeschool/sanfrancisco org team where mentions allow everyone to get notified at once.

**Comment by TheAlphaNerd** [comment]
+1 for not bringing in slack. I have so many, and total slack fatigue. It is a nice to have, but in the end it isn't really scalable.

**Comment by llkats** [comment]
Also, from the beginning, we've tried to make Nodeschool SF as transparent as possible, which is why we have relied on GitHub Issues for communication. Yes, it lacks the back-and-forth chat quality, but it is also all archived and searchable for posterity.

I'm definitely open to new ideas, but am also at Peak Slack Saturation myself.

**Comment by isurfbecause** [comment]
Sounds good guys, I understand :+1:

**Comment by jedireza** [comment]
It would be great if mentors arrived before 1pm (12:30 - 12:45 ish). See y'all wonderful people tomorrow.

**Comment by Trott** [comment]
Anyone have any idea if getting to Mozilla will be complicated by Super Bowl City or whatever it is they have downtown?

**Comment by LizzHale** [comment]
@Trott I work on 2nd and Brannan and commute in from Marin via the ferry (at the embarcadero) and/or a bus that drops off at 2nd and Harrison. It's been getting increasingly busy and yesterday my bag was searched before getting on the ferry. If Super Bowl City is open, I'd say it will complicate things but might not make it impossible for people to get in.

**Comment by Trott** [comment]
@LizzHale Thanks! My usual means of getting to Mozilla is BART to Embarcadero, but of course, I haven't done that since last year....

**Comment by TheAlphaNerd** [comment]
I didn't think about football... ugh.

✨

Gotta just make the best of it I guess

**Comment by llkats** [comment]
If you're taking BART to Embarcadero, I'd suggest transferring to the Muni N or T line underground and taking it the two or three stops to Moz. A lot of Market is completely closed or swarmed from Beale to the Ferry Building. South of the Ferry Building starts getting better, but there are still the weird street closures to deal with.

![muni/pedestrian map](https://www.sfmta.com/sites/default/files/pictures/2016/SB50%20Ped-Bike%20Map%201.15.16.png)

Instead of biking down Market, I've been heading down Mission and then going north to the FiDi for work; Mission seems pretty okay, and Folsom/Howard should be doable as well.

Broke-Ass Stuart had a good take.

![](http://brokeassstuart.com/wp-content/pictsnShit/2016/01/Muni-Map-sb-Finalz.jpg)
